src/java/org/endlessos/FullScreen.java

```diff
@@ -0,0 +1,87 @@
+package org.endlessos;
+
+import android.app.Activity;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.view.View;
+import android.webkit.WebChromeClient;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import android.widget.FrameLayout;
+import org.kivy.android.PythonActivity;
+
+public class FullScreen {
+    public PythonActivity mActivity;
+    public WebView mWebView;
+    public MyChrome mChrome;
+
+    public FullScreen(PythonActivity activity) {
+        mActivity = activity;
+        mWebView = (WebView) activity.getLayout().getChildAt(0);
+        mChrome = new MyChrome(activity);
+    }
+
+    public static void configureWebview(PythonActivity activity) {
+        FullScreen fs = new FullScreen(activity);
+        fs.configure();
+    }
+
+    // Configure the WebView to allow fullscreen based on:
+    // https://stackoverflow.com/questions/15768837/playing-html5-video-on-fullscreen-in-android-webview/56186877#56186877
```

Comment:
This suggests that we need updates to the `AndroidManifest.xml` as well, or does the P4A generated manifest already include the appropriate keys?

Reply:
In my tests the fullscreen was working correctly with just this java class, I didn't need to do anything else to have the fullscreen working for videos.